benchmarks/run.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+cd "$(dirname "${BASH_SOURCE[0]}")" || exit
+
+# Check that Hyperfine is installed.
+if ! command -v hyperfine > /dev/null 2>&1; then
+	echo "'hyperfine' does not seem to be installed."
+	echo "You can get it here: https://github.com/sharkdp/hyperfine"
+	exit 1
+fi
+
+rm -rf results/
+mkdir results/
+
+hyperfine \
+    --warmup=5 \
+    --time-unit=millisecond \
+    --export-json results/startup.json \
+    --export-markdown results/startup.md \
+    --command-name "Startup time (insect '1+1')" \
+    "node ../index.js '1+1'"
+
+hyperfine \
+    --warmup=5 \
+    --time-unit=millisecond \
+    --export-json results/startup-interactive.json \
+    --export-markdown results/startup-interactive.md \
+    --command-name "Startup time, interactive mode (insect < computations-1.ins)" \
```

Comment:
Is this not the non-interactive mode?

Reply:
Yes, "interactive" was poor naming. What I meant was: the mode that actually reads from STDIN, as compared to the one which performs just a single computation (`insect <EXPR>`). So maybe: "REPL mode".

Reply:
I think "stdin mode" is the clearest name.

Reply:
Done.